spec/_output/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
Same comment on this one; can we move the rule to the root `.gitignore` file? 
This will help us reduce maintenance complexity.